recipe/meta.yaml

```diff
@@ -17,28 +17,26 @@ build:
 requirements:
   host:
     - pip
-    - python
+    - python >=3.6
   run:
     - certifi
     - chardet ==3.*
     - h11 >=0.8,<0.10
     - h2 ==3.*
     - hstspreload
     - idna ==2.*
-    - python
+    - python >=3.6
     - rfc3986 >=1.3,<2
     - sniffio ==1.*
     - urllib3 ==1.*
 
 test:
   imports:
     - httpx
-    - httpx/_backends
-    - httpx/_dispatch
   requires:
-    - pytest
+    - pip
   commands:
-    - pytest --pyargs sixtools.tests
```

Comment:
What was wrong with `pytest`?

Reply:
From here https://github.com/conda-forge/httpx-feedstock/tree/master/recipe

Before I use the meta.yaml from pypi. But it has some error in CI, now, this one is from condda-forge.

Reply:
What about the removed pytest command?

Reply:
Ah, I see, the pytest is not a part of the recipe on CF. Looks good.